README.md

```diff
@@ -723,6 +724,44 @@ consumer.each_message do |message|
 end
 ```
 
+#### Customizing Partition Assignment Strategy
+
+In some cases, you might want to assign more partitions to some consumers. For example, in applications inserting some records to a database, the consumers running on hosts nearby the database can process more messages than the consumers running on other hosts.
+You can implement a custom assignment strategy and use it by passing an object that implements `#protocol_name`, `#user_data`, and `#assign` as the argument `assignment_strategy` like below:
+
+```ruby
+class CustomAssignmentStrategy
+  def initialize(user_data)
```

Comment:
```suggestion
  def initialize(cluster, user_data)
```

Reply:
I forgot to delete `@cluster` 💦 
I deleted the variable and force-pushed (https://github.com/zendesk/ruby-kafka/compare/34d69288a22d5593826c29ec8b787221efe237de..8e92d34b545b819b89595fbeafcb756b384bfea6).

![image](https://user-images.githubusercontent.com/508822/86536043-8e450200-bf1f-11ea-9b2f-0685e4e91229.png)